deployable/module-template/version.tf

```
@@ -0,0 +1,10 @@
+terraform {
+  required_version = ">= 1.3.0, <1.7.0"
+  # Lock the deployable module into an exact provider version - renovate automation will keep it updated
```

```suggestion
  # Lock the deployable module to a specific provider version. The Renovate automation updates the version as they are released.
```

or `as they are supported` ?